packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -7,7 +7,7 @@ import Timestamp from '@bbc/psammead-timestamp';
 import notes from '../README.md';
 import StoryPromo, { Headline, Summary, Link } from './index';
 
-const ImageComponent = (
+const buildImageComponent = () => (
```

Comment:
Could we define `ImageComponent` close to where it is actually used in the `story` instead of creating an extra function at the top to build the component.